vllm/worker/model_runner.py

```diff
@@ -62,8 +62,11 @@
 # Capture graphs for token size 1, 2, 4, 8, 16, 24, 32, 40, ..., 256.
 # NOTE: _get_graph_batch_size needs to be updated if this list is changed.
 _BATCH_SIZES_TO_CAPTURE = [1, 2, 4] + [
-    _BATCH_SIZE_ALIGNMENT * i for i in range(1, 33)
+    _BATCH_SIZE_ALIGNMENT * i for i in range(1, 34)
```

Comment:
can we directly extend this to `128`, i.e. the max batchsize to capture becomes 1024?

Reply:
You want to excused 768? Is there any perf implications for padding 768 up to 1024 when the max seq=768?

Reply:
we already use memory pool sharing for cudagraph, and the cost should only scale with the max batchsize.

if you capture batchsize 768, it has the same cost of capturing 760, 752,... together.